bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -3671,21 +3671,21 @@ void ReplicaImp::onMessage<ReqMissingDataMsg>(ReqMissingDataMsg *msg) {
       }
     }
 
-    if (msg->getFullCommitIsMissing()) {
+    if (msg->getFullCommitIsMissing() && seqNumInfo.isTimeCorrect()) {
       CommitFullMsg *c = seqNumInfo.getValidCommitFullMsg();
```

Comment:
the commit full message should be nullptr in the seq num info if time is not correct.